docs/client/jdbc/kyuubi_jdbc.rst

```diff
@@ -147,6 +147,28 @@ Connection URL over Service Discovery
 - zookeeper quorum is the corresponding zookeeper cluster configured by `kyuubi.ha.addresses` at the server side.
 - zooKeeperNamespace is  the corresponding namespace configured by `kyuubi.ha.namespace` at the server side.
 
+HiveServer2 Compatibility
+*************************
+
+.. versionadded:: 1.8.0
```

Comment:
1.8.0 rc0 is already in voting and will be released soon.

Reply:
rc1 is coming ...